services/viva-ms/src/helpers/completions.js

```diff
@@ -78,9 +110,23 @@ async function getCaseOnWorkflowId(personalNumber, workflowId) {
   return caseItem;
 }
 
-function isCaseStateCompletions(caseItem) {
-  const { state } = caseItem;
-  return COMPLETIONS_TYPE.reduce((all, type) => state.includes(type) || all, false);
+function getLocaleDate(value) {
+  const LOCALE_OPTION = {
+    timeZone: 'Europe/Stockholm',
+  };
+
+  return new Date(new Date(value).toLocaleDateString('sv-SE', LOCALE_OPTION)).setHours(0, 0, 0, 0);
+}
+
```

Comment:
This function only uses sv-SE as locale, but to make it further usable I suggest adding locale as parameter instead. And also, parameter `value` isn't really a good parameter name, I suggest renaming it to date or similar. 

```js
function getLocaleDate(date, locale) {
  return new Date(date).toLocaleDateString(locale).setHours(0, 0, 0, 0);
}
```

or just renaming the function (and renaming the function):
```js
function getSvLocaleDate(date) {
  return new Date(date).toLocaleDateString('sv-SE').setHours(0, 0, 0, 0);
}
```

also, feels like `livrem och hängslen` when doing new Date(new Date))...  the constructor for new Date takes a string or number so you should be fine with above change.

... and one more thing.
If i'm not mistaken, Sweden only have one timezone, so adding the timezone might not be needed.